test/Intrinsics/fabs.ll

```diff
@@ -19,10 +19,6 @@
 ;  return 0;
 ;}
 ; ```
-; To clean the resulting llvm-ir up for LLVM-versions < 6,
```

Comment:
Please keep this comment, it's about how this file was generated in the first place.